### PR TITLE
Port to jetty 10.x and update deps to latest

### PR DESCRIPTION
**Pull request by ghaskins**
We intentionally hold back on tomcat 10.x as there is more porting work than to simply
bump the dependencies up.

It is suggested that if this is merged, the minor version should be bumped before the next release.

Signed-off-by: Greg Haskins <greg@manetu.com>

### CODE REVIEW

**Comment by hlship** [comment]
Thank you for contributing to Pedestal!
    
Sorry it's taken us such a log time to get around to addressing your Pull Request. We'd like to integrate your contribution into the project, but in order to do this we need to get a signed Contributor Agreement from you.
lease find instructions on how (and why) to do this here:
    
http://pedestal.io/community/contributing#_contributors_agreement
    
Thanks!

**Comment by ghaskins** [comment]
CLA signed

File: service/project.clj [hunk 1]
```diff
@@ -16,31 +16,34 @@
...
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.10.1"]
+  :dependencies [[org.clojure/clojure "1.11.1"]
 
                  [io.pedestal/pedestal.log "0.5.11-SNAPSHOT"]
                  [io.pedestal/pedestal.interceptor "0.5.11-SNAPSHOT"]
                  [io.pedestal/pedestal.route "0.5.11-SNAPSHOT"]
 
+                 [org.ow2.asm/asm "9.3"]
```
Comment (souenzzo) [inline service/project.clj:25]: Why does it is required?

Comment (ghaskins) [reply service/project.clj:25]: I actually no longer recall.  It was probably either one of two things, both related to transitive deps:

1) needed to be updated for the PR to work
2) there was a CVE 

IIRC, it was the former (1).  I can try to look later.

**Comment by hlship** [comment]
Replaced by #762

**Pull request closed without merging**